src/coreclr/gc/gc.cpp

```diff
@@ -6943,6 +6959,25 @@ bool gc_heap::virtual_commit (void* address, size_t size, int bucket, int h_numb
 
     dprintf(3, ("commit-accounting:  commit in %d [%p, %p) for heap %d", bucket, address, ((uint8_t*)address + size), h_number));
 
+#ifdef COMMITTED_BYTES_SHADOW
```

Comment:
let's try to not have duplicated code here and write it in the following block to update the appropriate fields